x/deployment/types/validation_config.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+// This is the validation configuration that acts as a hard limit
+// on what the network accepts for deployments. This is never changed
+// and is the same across all members of the network
+
+var validationConfig = struct {
+	MaxUnitCPU     uint
+	MaxUnitMemory  uint
+	MaxUnitStorage uint
+	MaxUnitCount   uint
+	MaxUnitPrice   uint
+
+	MinUnitCPU     uint
+	MinUnitMemory  uint
+	MinUnitStorage uint
+	MinUnitCount   uint
+	MinUnitPrice   uint
+
+	MaxGroupCount int
+	MaxGroupUnits int
+
+	MaxGroupCPU     int64
+	MaxGroupMemory  int64
+	MaxGroupStorage int64
+
+	MinGroupMemPrice int64
+	MaxGroupMemPrice int64
+}{
+	MaxUnitCPU:     500,
```

Comment:
I'm bleary eyed, but a little confused on this MaxUnitCPU. Is this supposed to be 500 CPU cores? Otherwise this would translate into 500milliCPUs(kubernetes-ism), and thus not a lot of processing power.

Reply:
good point I don't remember myself.

Reply:
I will double check this, I don't know as I just refactored this code.

Reply:
Okay, I think this is counted in milliCPU, so 1000 = 1CPU core. So MaxUnitCPU might best be 64000?

Reply:
OK, I'm changing it to 64000

Reply:
I think it was actually intentional - we only wanted to allow half a cpu share at the time.  Let's cap it to, I don't know, 1000 = 1 cpu share for now - this is for a single instance, 64 is too large IMHO; you'd need a box with 64 unused cpus to fulfill it.

Reply:
Ok I moved it to 1000